src/libraries/Microsoft.Extensions.Configuration.Xml/src/XmlConfigurationElementContent.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Extensions.Configuration.Xml
+{
+    internal class XmlConfigurationElementContent
+        : IXmlConfigurationValue
+    {
+        private readonly XmlConfigurationElement[] _elementPath;
+
+        public XmlConfigurationElementContent(Stack<XmlConfigurationElement> elementPath, string content, string lineInfo)
+        {
+            Value = content ?? throw new ArgumentNullException(nameof(content));
+            LineInfo = lineInfo ?? throw new ArgumentNullException(nameof(lineInfo));
+            _elementPath = elementPath?.Reverse().ToArray() ?? throw new ArgumentNullException(nameof(elementPath));
```

Comment:
Perhaps you could only save the keys since that is all you need in line 27 (for Key)

Reply:
@maryamariyan Please see my comment here: https://github.com/dotnet/runtime/pull/44608#issuecomment-748182186

I believe this comment is now outdated.